### DISCUSSION TITLE
Load entities

### DISCUSSION THREAD

**Discussion by vincenzomazzeo** [gicamm/homeassistant-comelit#64] [General]
Hello @gicamm ,

First of all thank you very much for your work!

I have a question related to load entities.

I see that in JSON are present sections regarding loads like the following:

```json
{
  "id": "DOM#LC#2.1",
  "data": {
    "id": "DOM#LC#2.1",
    "type": 10,
    "sub_type": 0,
    "descrizione": "DISH WASHER",
    "sched_status": "0",
    "status": "0",
    "powerst": "0",
    "num_modulo": 2,
    "num_uscita": 1,
    "icon_id": "0",
    "objectId": "DOM#LC#2.1",
    "placeId": "",
    "toolbar_active": "0",
    "instant_power": "176",
    "num_module_TA": 8,
    "num_out_TA": 1,
    "description_TA": "Controllo carichi TA ",
    "out_power": 2
  }
}
```

Are they supposed to be visible in HA?

Thank you in advance for the response!

**Comment by gicamm** [comment]
Hello Vincenzo,
At the moment there are no loads managed.
But it's a nice thing to have.
